go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/go-debos/debos
+
+go 1.15
+
+require (
+	github.com/docker/go-units v0.4.0
+	github.com/go-debos/fakemachine v0.0.0-20220412175549-8719c1b0b5bb
```

Comment:
time to properly tag a fakemachine release?

Reply:
not quite; but i think in april we will do so ;-)